src/lib/components/Swap/Summary.fixture.tsx

```diff
@@ -29,13 +35,18 @@ function Fixture() {
       [Field.OUTPUT]: UNI,
     })
     setInitialized(true)
-  })
+  }, [setState])
 
-  return initialized ? (
+  return initialized && trade ? (
```

Comment:
Remove `initialized` entirely (including L23) and condition this on `trade` alone.